using-mobile-extensions/adobe-campaign-standard/adobe-campaign-standard-event-reference.md

```diff
@@ -0,0 +1,178 @@
+
+| **Key** | **Value Type** | **Optional** | **Description** |
+| :--- | :--- | :--- | :--- |
+| `__dev__campaign.server` | String | Yes | This contains the endpoint URL for the development environment Adobe Campaign Standard instance |
+| `__dev__campaign.pkey` | String | Yes | This contains the identifier for a mobile app configured in development environment Adobe Campaign Standard |
+| `__stage__campaign.server` | String | Yes | This contains the endpoint URL for the stage environment Adobe Campaign Standard instance |
+| `__stage__campaign.pkey` | String | Yes | This contains the identifier for a mobile app configured in stage environment Adobe Campaign Standard |
+| `campaign.server` | String | Yes | This contains the endpoint URL for the production environment Adobe Campaign Standard instance |
+| `campaign.pkey` | String | Yes | This contains the identifier for a mobile app configured in production environment Adobe Campaign Standard |
+| `campaign.mcias` | String | Yes | This contains the in-app messaging service URL endpoint |
+| `campaign.timeout` | String | Yes | This contains the amount of time to wait for a response from in-app messaging service |
+| `global.privacy` | Boolean | Yes | This contains the mobile privacy status settings |
+
+{% hint style="info" %}
+If `global.privacy` is changed to **optout**, linkage fields will be reset. All downloaded messages, rules will be erased and no tracking request may leave the device.
+{% endhint %}
+
+#### Event data example
+
+```text
+{
+    "global.privacy":"optedin",
+    "campaign.timeout":1,
+    "campaign.mcias":"mcias-va7.cloud.adobe.io/mcias",
+    "__dev__campaign.pkey":"",
+    "__stage__campaign.pkey":"",
+    "campaign.pkey":"",
+    "__dev__campaign.server":"mcias.campaign-demo.adobe.com",
+    "__stage__campaign.server":"mcias.campaign-demo.adobe.com",
+    "campaign.server":"mcias.campaign-demo.adobe.com"
+}
+```
+
+### Request Identity  <a id="request-identity"></a>
+
+This event is dispatched by public API to setLinkageFields to set the linkage fields. It is used to download personalized In-App messages.
+
+#### Data payload definition  <a id="data-payload-definition-4"></a>
+
+Here are the key-value pairs in this event:
+
+| **Key** | **Value Type** | **Optional** | **Description** |
+| :--- | :--- | :--- | :--- |
+| `linkage_fields` | Map<String, String> | No | This map contains either one or multiple linkage fields that are used to personally identify the logged in user. |
+
+#### Event data example
+
+```text
+{ 
+    "linkagefields": 
+    { 
+        "firstName":"john", 
+        "lastName":"doe", 
+        "age":30
+    }
+}
+```
+
+### Request Reset  <a id="request-reset"></a>
+
+This event is dispatched when a user decides they no longer wants to utilize personalized In-App messages. Following the recieval of this event, campaign deletes all personalized messages and redownloads only generic messages.
```

Comment:
Please reword as follows:

"This event is dispatched when users no longer want to use personalized in-app messages. After this event is received, Campaign deletes all personalized messages and downloads only generic messages again."